cpp/unit/unit.cpp

```diff
@@ -106,16 +107,20 @@ void Unit::update_secondary(int64_t time_elapsed) {
 }
 
 void Unit::draw() {
+	this->draw(this->location, this->graphics);
```

Comment:
Uhm, doesn't the second line in `Unit::draw` say `auto &draw_pos = loc->pos.draw;`?